js/spa/test/cancel_events.spec.ts

```diff
@@ -7,20 +7,19 @@ test("when using an iterative function the UI should update over time as iterati
 	const start_button = await page.locator("button", {
 		hasText: /Start Iterating/
 	});
-	const textbox = await page.getByLabel("Iterative Output");
```

Comment:
Why this change?

Reply:
I thought it was related to the e2e changes but it's not!